README.md

```diff
@@ -2,7 +2,7 @@
 
 ![GitHub tag (latest SemVer)](https://img.shields.io/github/v/tag/synopsys-sig/synopsys-action?color=blue&label=Latest%20Version&sort=semver)
 
-The latest version of the Synopsys Bridge is: [Version 0.7.2](https://sig-repo.synopsys.com/artifactory/bds-integrations-release/com/synopsys/integration/synopsys-action/0.1.72/)
+The latest version of the Synopsys Bridge is: [Version 0.1.114](https://sig-repo.synopsys.com/artifactory/bds-integrations-release/com/synopsys/integration/synopsys-action/0.1.114/)
```

Comment:
is it required to specify because if there is another upgrade on bridge, we need to change our readme?
Also we need to specify in readme that action is capable of downloading automatically latest version of bridge.

Reply:
In that case, we can provide link to just bridge repository. Let me know, if that will be okay

Reply:
yes. The latest version of the Synopsys Bridge is available at:

Reply:
Done